src/communication/README.md

```diff
@@ -0,0 +1,56 @@
+# Azure CLI communication Extension #
+This is the extension for communication
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name communication
+```
+
+### Included Features ###
+#### communication communication-service ####
```

Comment:
Not compatible with command group description.
aligning with `az communication service` will be better.

Reply:
Fixed